internal/pubsub/example_test.go

```diff
@@ -28,11 +28,10 @@ import (
 func Example_sendReceive() {
 	// Open a topic and corresponding subscription.
 	ctx := context.Background()
-	dt := mempubsub.OpenTopic()
-	ds := mempubsub.OpenSubscription(dt, time.Second)
-	t := pubsub.NewTopic(dt)
+	b := mempubsub.NewBroker([]string{"t"})
```

Comment:
Nit: for an example, I suggest using a more descriptive name, like "MyTopic" or something like that.

Reply:
Push?

Reply:
Waiting for resolution on the other comment before I push. I guess I should not have commented 'Done' just because I made the change locally.

Reply:
GO ahead and push; we can update the code later to match #759 if needed.

Reply:
pushed